packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -445,6 +445,15 @@ export default class Dropdown extends React.Component<DropdownProps, State> {
         }
     };
 
+    getItemRole() {
+        switch (this.props.role) {
+            case "listbox":
+                return "option";
+            case "menu":
+                return "menuitem";
+        }
```

Comment:
NIT: Should have a "default" case to throw an error if the role isn't one we explicitly support. Will make maintenance easier if things change expectedly or otherwise.

Reply:
I'll make that change before merging.

Reply:
While I added a throw here, I did not write a test for it because I would've had to disable flow checking to write a test that actually throws which seemed a little silly.

Reply:
Oh, I just cast to `any` to get around that. :)